### PR TITLE
fix: typo CustomConcurrenyStrategy to CustomConcurrencyStrategy

### PR DESCRIPTION
**Pull request by typekpb**

### CODE REVIEW

**Comment by pivotal-issuemaster** [comment]
@typekpb Please sign the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-openfeign&pullRequestId=100)!

[Click here](https://cla.pivotal.io/sync/spring?repositoryId=spring-cloud/spring-cloud-openfeign&pullRequestId=100) to manually synchronize the status of this Pull Request.

See the [FAQ](https://cla.pivotal.io/about) for frequently asked questions.

File: spring-cloud-openfeign-core/src/test/java/org/springframework/cloud/openfeign/hystrix/security/HystrixSecurityApplication.java [hunk 1]
```diff
@@ -38,8 +38,8 @@
 public class HystrixSecurityApplication {
 
 	@Bean
-	public CustomConcurrenyStrategy customConcurrenyStrategy() {
-		return new CustomConcurrenyStrategy();
+	public CustomConcurrencyStrategy CustomConcurrencyStrategy() {
```
Comment (ryanjbaxter) [inline spring-cloud-openfeign-core/src/test/java/org/springframework/cloud/openfeign/hystrix/security/HystrixSecurityApplication.java]: method name should not be capitalized

**Comment by ryanjbaxter** [comment]
Actually this doesnt even compile because you didnt change the file name.

**Comment by OlgaMaciaszek** [comment]
Closing as no fix provided.

**Comment by pivotal-issuemaster** [comment]
@typekpb Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-openfeign&pullRequestId=100)!

**Comment by pivotal-issuemaster** [comment]
@typekpb Thank you for signing the [Contributor License Agreement](https://cla.pivotal.io/sign/spring?repositoryId=spring-cloud/spring-cloud-openfeign&pullRequestId=100)!

**Pull request closed without merging**